frame/support/procedural/src/lib.rs

```diff
@@ -325,3 +326,8 @@ pub fn construct_runtime(input: TokenStream) -> TokenStream {
 pub fn transactional(attr: TokenStream, input: TokenStream) -> TokenStream {
 	transactional::transactional(attr, input).unwrap_or_else(|e| e.to_compile_error().into())
 }
+
+#[proc_macro]
+pub fn crate_to_pallet_version(input: TokenStream) -> TokenStream {
```

Comment:
brief doc please :).

Reply:
The docs are in frame support. I can not put them here.